source/docs/reference/server-timings.md

```diff
@@ -0,0 +1,32 @@
+---
+id: server-timings
+title: Server Timings
+---
+
+[Server Timings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing) are metrics that are sent via a header in the response of an HTTP request. They allow you to know which part is slowing a request from your server.
+
+You can see a representation in your network panel of your devtools by clicking on request and opening the "Timings" tab. Follow these guides for [Firefox](https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor/request_details#Timings)'s or [Google Chrome](https://www.smashingmagazine.com/2018/10/performance-server-timing/#the-server-timing-header)'s devtools.
+
+The Server Timings are enabled when one of the following condition is satisfied:
+
+* `FRONT_COMMERCE_ENV !== "production"`
+* `DEBUG=front-commerce:performance`
```

Comment:
We should also add it to https://deploy-preview-109--elastic-austin-cf9d40.netlify.com/docs/reference/environment-variables.html#Debugging